### PR TITLE
Migrate asana-get-automation-subtask-id from GHA action to fastlane

### PR DESCRIPTION
**Pull request by ayoy**
Task/Issue URL: https://app.asana.com/0/0/1208137627434493/f

**Description**:
This change removes asana-get-automation-subtask-id action, replacing it with a fastlane action from the automation plugin.

**Steps to test this PR**:
1. Verify that CI is green.
2. Verify that [this test workflow](https://github.com/duckduckgo/macos-browser/actions/runs/10627897390/job/29461773765) completes successfully. Verify that [this Asana task](https://app.asana.com/0/0/1208137627434473/f)'s Automation subtask has 1 new comment and 1 new subtask. This workflow is here only to demonstrate that the fastlane action works.
3. Review [fastlane plugin](https://github.com/duckduckgo/fastlane-plugin-ddg_apple_automation/pull/1) PR.

**Definition of Done**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: .github/workflows/migration_test_asana_get_automation_subtask_id.yml
Comment (ayoy) [inline .github/workflows/migration_test_asana_get_automation_subtask_id.yml:1]: This file will be removed before merging :)

File: .github/actions/asana-log-message/action.yml [hunk 1]
```diff
@@ -24,10 +24,10 @@ runs:
   using: "composite"
   steps:
     - id: get-automation-subtask
-      uses: ./.github/actions/asana-get-release-automation-subtask-id
-      with:
-        access-token: ${{ inputs.access-token }}
-        task-url: ${{ inputs.task-url }}
+      shell: bash
```
Comment (jaceklyp) [inline .github/actions/asana-log-message/action.yml:27]: is this needed?

**Review by jaceklyp — APPROVED** [review]
LGTM :)

File: .github/actions/asana-log-message/action.yml (hunk 1 above)
Comment (ayoy) [reply .github/actions/asana-log-message/action.yml:27]: Yes, shell definition is needed for actions steps.

**Pull request merged**